app/assets/javascripts/discourse/app/templates/components/sidebar.hbs

```diff
@@ -10,5 +10,7 @@
     {{#if this.siteSettings.enable_personal_messages}}
       <Sidebar::MessagesSection />
     {{/if}}
+
+    <PluginOutlet @name="after-sidebar" />
```

Comment:
Let's leave a comment here to say that this should not be used and will be removed in the very very near future.